qa/smoke-test-ingest-with-all-dependencies/src/test/java/org/elasticsearch/ingest/IngestDocumentMustacheIT.java

```diff
@@ -31,7 +31,7 @@
 
 import static org.hamcrest.Matchers.equalTo;
 
-public class IngestDocumentMustacheIT extends AbstractMustacheTests {
+public class IngestDocumentMustacheIT extends AbstractMustacheTestCase {
```

Comment:
On second thought lets do that in another PR. Its a nice win just to run the checks for now.